index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>GC Logger Local Test Page</title>
+</head>
+
+<body>
+
+  <script src="dist/genesys-cloud-client-logger.js"></script>
+  <script>
+    const accessToken = 'INSTER_TOKEN_HERE';
+
+    window.logger = new GenesysCloudClientLogger({
+      accessToken,
+      url: 'https://api.inindca.com/api/v2/diagnostics/trace',
+      appVersion: '2.0.0',
+      logTopic: `gc-client-logger-testing`,
+      logLevel: 'info',
+      uploadDebounceTime: 2500
+    });
+
+
+    fetch('https://api.inindca.com/api/v2/tokens/me', {
+      headers: {
+        Authorization: `Bearer ${accessToken}`
+      },
+      method: 'get'
+    })
+      .then(d => d.json())
+      .then(d => console.log('response', d))
```

Comment:
NIT: this console.log doesn't need its own `.then`

Reply:
I'm going to delete this before merging I think